<launch>
  <arg name="camera_side" default="left" />
  <arg name="marker_size" default="5.8" />
  <arg name="max_new_marker_error" default="0.15" />
  <arg name="max_track_error" default="0.3" />
  <arg name="cam_image_topic" default="/cameras/$(arg camera_side)_hand_camera/image" />
  <arg name="cam_info_topic" default="/cameras/$(arg camera_side)_hand_camera/camera_info" />
  <arg name="output_frame" default="/$(arg camera_side)_hand" />

  <node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect"
              respawn="true" output="log"
              args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error)
                    $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />
  <node name="ar_track_alvar_board" pkg="ar_track_alvar" type="findMarkerBundlesNoKinect"
        respawn="true" output="log"
        args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error)
              $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)
              $(find tetris)/bundles/board.xml"/>
</launch>
